### DISCUSSION TITLE
Can't I do a redirect from "/"?

### DISCUSSION THREAD

**Discussion by mastoj** [vercel/next.js#46065] [Help] [answered]
It seems like I can't do redirects from "/" to another url? Is that correct or do I have to do something else to do it than just using redirect in the next.config.js?

**Comment by icyJoseph** [comment, marked as answer]
Could you give an example of what you've tried, next version? I tested with:

```js
/** @type {import('next').NextConfig} */
const nextConfig = {
  reactStrictMode: true,
  async redirects() {
    return [
      {
        source: "/",
        permanent: false,
        destination: "/api/hello",
      },
    ];
  },
};

module.exports = nextConfig;
```

No issues. Using an absolute path as `destination` worked fine too.

**Comment by andorthehood** [comment]
> It seems like I can't do redirects from "/" to another url? Is that correct or do I have to do something else to do it than just using redirect in the next.config.js?

I had the same problem, I had to delete the index file from the root of the paths folder to make this redirect work.

**Reply by icyJoseph** [reply]
But that shouldn't be necessary, could you try to make a stackblitz example or share some code?

**Comment by mastoj** [comment]
For me it turned out to be a stupid mistake. I already had some rewrite rules, and didn't reflect over the fact that it said rewrite and not redirect. Changing to redirect worked like charm.

**Reply by icyJoseph** [reply]
Glad you managed to fix it! Don't forget to mark it as resolved.
